### PR TITLE
[WIP] Create amp-consent modal UI example

### PR DESCRIPTION
**Pull request by micajuine-ho**
Show an example of how to use new consent modal UI through amp-consent.

### CODE REVIEW

**Comment by patrickkettner** [comment]
@micajuine-ho did you have any thoughts?

**Comment by micajuine-ho** [comment]
@patrickkettner Thanks for the bump. 

I am having trouble getting the `promptUiSrc` iframe to show up on the page. Basically, it can't find the file, even thought I added it. I'm probably referencing it wrong within the config. Would you or @sebastianbenz  have any ideas on how to fix this?

File: examples/source/user-consent/User_Consent_Modal_UI.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+<body>
+  <!-- ## Defining the Consent Flow -->
+  <!--
+    We define a basic consent flow using the `promptUISrc` to load our custom consent iframe.
+  -->
+  <amp-consent id="myUserConsent" layout="nodisplay">
+    <script type="application/json">{
+      "consentInstanceId": "myConsent",
+      "consentRequired": true,
+      "promptUISrc": "/examples/static/samples/files/diy-consent.html",
```
Comment (micajuine-ho) [inline examples/source/user-consent/User_Consent_Modal_UI.html:50]: I can see it making the request in the network tab, but it's saying that the file can't be found.

**Comment by sebastianbenz** [comment]
@micajuine-ho sorry for dropping the ball on this. I've fixed the path.

**Comment by micajuine-ho** [comment]
> @micajuine-ho sorry for dropping the ball on this. I've fixed the path.

No worries, did you get a chance to verify it?

**Comment by sebastianbenz** [comment]
Yes, the `/examples` prefix needed to be removed.

>

**Pull request merged**